### PR TITLE
CU-1ej8fr6 Add Fang Tutorial & Complete Intro scene

### PR DESCRIPTION
**Pull request by FrancPS**
Note: future Onimaru ultimate rebalance will fit correctly with the encounter difficulty on that part of the tutorial.

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-1ej8fr6 Add Fang Tutorial](https://app.clickup.com/t/1ej8fr6)

**Review by isaacfdez — DISMISSED** [review]

**Review by Jordi-Gil — APPROVED** [review]
The key to continue the dialogues is SPACE, but in the image still showing F

File: Penteract/Assets/Scripts/DialogueManager.cpp [hunk 1]
```diff
@@ -106,22 +107,21 @@ void DialogueManager::Start() {
...
 
 	// LEVEL 1 - FANG TUTORIAL
-	dialoguesArray[15] = Dialogue(DialogueWindow::ONIMARU, true, "He is running away!\nDon't let him escape!", &dialoguesArray[16]);
-	dialoguesArray[16] = Dialogue(DialogueWindow::TUTO_FANG, true, "", &dialoguesArray[17]);
-	dialoguesArray[17] = Dialogue(DialogueWindow::TUTO_FANG_ULTI, true, "", nullptr);
+	dialoguesArray[15] = Dialogue(DialogueWindow::TUTO_FANG, true, "", &dialoguesArray[16]);
+	dialoguesArray[16] = Dialogue(DialogueWindow::TUTO_FANG_ULTI, true, "", nullptr);
 
 	// LEVEL 1 - SWAP DIALOGUE + ONIMARU TUTORIAL
 	dialoguesArray[18] = Dialogue(DialogueWindow::FANG, true, "Onimaru,\nget the repair bots\nready...\nI'm gonna need a break.", &dialoguesArray[19]);
```
[the remark this answers is not in the document]
Comment (FrancPS) [reply Penteract/Assets/Scripts/DialogueManager.cpp:122]: It's not a problem

**Comment by FrancPS** [comment]
> 
> 
> The key to continue the dialogues is SPACE, but in the image still showing F

This will be edited when the feature is passed to level1.scene

**Review by isaacfdez — DISMISSED** [review]

**Review by magalenyo — CHANGES_REQUESTED** [review]
When pressing Reload Scene on the Main Menu, after video plays, can't perform any action and no Dialogue appears until you open and close the menu. This is probably due to Pause's stop time behavior. This also happens when comming from the StartScene.

**Comment by Jordi-Gil** [comment]
> When pressing Reload Scene on the Main Menu, after video plays, can't perform any action and no Dialogue appears until you open and close the menu. This is probably due to Pause's stop time behavior. This also happens when comming from the StartScene.

When comming from the StartScene, is probably loading the Level1.scene not the Level1-Tutorial.scene

**Review by Jordi-Gil — CHANGES_REQUESTED** [review]
When you are in a dialogue and the game is stopped, and you press the ESC button to open the pause menu. Once you return to the game, you can move it with the dialogues actives.

92e1e15c207b925f44c9e688cdad480c

**Comment by FrancPS** [comment]
> 
> 
> When pressing Reload Scene on the Main Menu, after video plays, can't perform any action and no Dialogue appears until you open and close the menu. This is probably due to Pause's stop time behavior. This also happens when comming from the StartScene.

Checked. It works fine when the reload and start scene load Level1-Tutorial.scene instead of Level1.scene.

**Comment by FrancPS** [comment]
> 
> 
> When you are in a dialogue and the game is stopped, and you press the ESC button to open the pause menu. Once you return to the game, you can move it with the dialogues actives.

Fixed

**Comment by FrancPS** [comment]
I'm including CU-1f91bf7 in this PR. Hold for reviews.

**Pull request closed without merging**